browse/templates/home/home.html

```diff
@@ -53,9 +53,17 @@
     <h4 class="homepage-news-title">News</h4>
     <!-- special news section -->
     {%- include "home/news.html" -%}
-    Read about recent news and updates on <a href="https://blog.arxiv.org/" target="_blank">arXiv's blog</a>.
-    (View the former <a href="https://arxiv.org/new/">"what's new" pages</a> here).
-    Read <a href="/help/robots">robots beware</a> before attempting any automated download.
+    <div><b>There may be momentary outages of arxiv.org due to systems maintenance
+        happening at Cornell on Monday Nov. 14th 5am to 7am EST (10:00 to 12:00 UTC).
```

Comment:
```suggestion
        on Monday Nov. 14th 5am to 7am EST (10:00 to 12:00 UTC).
```

Just simplifying wording as we are Cornell.